frontend/api/books.js

```diff
@@ -1,92 +1,92 @@
-import { gql, useQuery, useMutation } from "@apollo/client";
+import { gql, useQuery, useMutation } from '@apollo/client';
 
-export const useGetBooks = ( ) => {
-  const allBooksQuery = gql`
-    query getBooks {
-      getBooks {
+export const allBooksQuery = gql`
+  query getBooks {
+    getBooks {
+      id
+      title
+      coverImage
+      author {
         id
-        title
-        coverImage
-        author {
-          id
-          firstName
-          lastName
-        }
+        firstName
+        lastName
+      }
+    }
+  }
+`;
+export const aBookQuery = gql`
+  query getBook($id: ID!) {
+    getBook(id: $id) {
+      id
+      title
+      coverImage
+      author {
+        id
+        firstName
+        lastName
       }
+      description
     }
-  `;
+  }
+`;
+
+export const useGetBooks = () => {
   const { loading, error, data } = useQuery(allBooksQuery);
 
   return {
     booksLoading: loading,
     booksError: error,
-    books: data?.getBooks || [],
+    books: data?.getBooks || []
   };
 };
 
-export const useGetBook = (id) => {
-  const query = gql`
-    query getBook($id: ID!) {
-      getBook(id: $id) {
-        id
-        title
-        coverImage
-        author {
-          id
-          firstName
-          lastName
-        }
-        description
-      }
-    }
-  `;
-
-  const { loading, error, data } = useQuery(query, { variables: { id } });
+export const useGetBook = id => {
+  const { loading, error, data } = useQuery(aBookQuery, { variables: { id } });
 
   return {
     bookLoading: loading,
     bookError: error,
-    book: data?.getBook || {},
+    book: data?.getBook || {}
   };
 };
 
+export const addBookMutation = gql`
```

Comment:
See export comment, love that you declared this mutation outside of the hook so that it isn't reinstanced on ever render, but just double checking that it's necessary for this export.

Reply:
only doing to support the jest test. perhaps i am going about that wrong (see above)?